.travis.yml

```diff
@@ -38,6 +47,10 @@ cache:
     - $HOME/.pyenv_cache
 
 before_install:
+  - | 
```

Comment:
What is this section for?

Reply:
This section is for testing multiline yml support.

Thanks !!